features/admins/admin_skins.feature

```diff
@@ -0,0 +1,114 @@
+  Then I should see "The following skins were updated: public skin"
+
+  Scenario: An admin can reject and unreject a skin
+  Given the unapproved public skin "public skin"
+    And I am logged in as an admin
+  When I go to admin's skins page
+    And I check "make_rejected_public_skin"
+    And I submit
+  Then I should see "The following skins were updated: public skin"
+  When I follow "Rejected Skins"
+  Then I should see "public skin"
+  When I check "make_unrejected_public_skin"
+    And I submit
+  Then I should see "The following skins were updated: public skin"
+  When I follow "Rejected Skins"
+  Then I should not see "public skin"
+
+  Scenario: An admin can feature and unfeature skin
+  Given the approved public skin "public skin"
+    And I am logged in as an admin
+  When I follow "Approved Skins"
+    And I check "Feature"
+    And I submit
+  Then I should see "The following skins were updated: public skin"
+  When I follow "Approved Skins"
+    And I check "Unfeature"
+    And I submit
+  Then I should see "The following skins were updated: public skin"
+
+  Scenario: Admins should be able to see public skins in the admin skins page
+  Given the unapproved public skin "public skin"
+    And I am logged in as an admin
+  When I go to admin's skins page
+  Then I should see "public skin" within "table#unapproved_skins"
+
+  Scenario: Admins should not be able to edit unapproved skins
+  Given the unapproved public skin "public skin"
+    And I am logged in as an admin
+  When I go to "public skin" skin page
+  Then I should not see "Edit"
+    And I should not see "Delete"
+  When I go to "public skin" edit skin page
+  Then I should see "Sorry, you don't have permission"
+
+  Scenario: Admins should be able to approve public skins
+  Given the unapproved public skin "public skin"
+    And I am logged in as an admin
+  When I go to admin's skins page
+    And I check "public skin"
+    And I submit
+  Then I should see "The following skins were updated: public skin"
+  When I follow "Approved Skins"
+  Then I should see "public skin" within "table#approved_skins"
+
+  Scenario: Admins should be able to edit but not delete public approved skins
+  Given the approved public skin "public skin" with css "#title { text-decoration: blink;}"
+    And I am logged in as an admin
+  When I go to "public skin" skin page
+  Then I should see "Edit"
+    But I should not see "Delete"
+  When I follow "Edit"
+    And I fill in "CSS" with "#greeting.logged-in { text-decoration: blink;}"
+    And I fill in "Description" with "Blinky love (admin modified)"
+    And I submit
+  Then I should see an update confirmation message
+    And I should see "(admin modified)"
+    And I should see "#greeting.logged-in"
+    And I should not see "#title"
+  Then the cache of the skin on "public skin" should expire after I save the skin
+
+  Scenario: Admins should be able to unapprove public skins, which should also
+  remove them from preferences
```

Comment:
If it works leave it as it is, I do think I prefer a blank line between the Scenario and the test if we do this.